shared/composemanager/compose_manager.go

```diff
@@ -47,7 +48,48 @@ func (c ComposeManager) WithHappyConfig(happyConfig *config.HappyConfig) Compose
 	return c
 }
 
-func (c ComposeManager) Generate(ctx context.Context) error {
+func (c ComposeManager) Manage(ctx context.Context) error {
+	composeFilePath := c.HappyConfig.GetBootstrap().DockerComposeConfigPath
+	if _, err := os.Stat(composeFilePath); err == nil || !errors.Is(err, os.ErrNotExist) {
```

Comment:
swap your error logic. If there is an error, bail and throw the error. Otherwise, do your logic

https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting